psl/psl/tests/validation_tests.rs

```diff
@@ -56,13 +87,36 @@ fn run_validation_test(test_file_path: &str) {
             .unwrap_or(source_file.as_str());
         file.write_all(schema.as_bytes()).unwrap();
 
-        for line in errors.lines() {
+        for line in diagnostics.lines() {
             writeln!(file, "// {line}").unwrap();
         }
         return;
     }
 
-    panic_with_diff::panic_with_diff(&last_comment_contents, &errors)
+    panic_with_diff::panic_with_diff(&last_comment_contents, &diagnostics)
+
+    // let errors = match (last_comment_contents.is_empty(), validation_result) {
+    //     (true, Ok(_)) => return, // expected and got a valid schema
+    //     (false, Err(errors)) if last_comment_contents == errors => return, // we expected the errors we got
+    //     (_, Err(errors)) => errors,
+    //     (false, Ok(_)) => String::new(), // expected errors, got none
+    // };
+
+    // if std::env::var("UPDATE_EXPECT").is_ok() {
+    //     let mut file = fs::File::create(&file_path).unwrap(); // truncate
+    //
+    //     let schema = last_comment_idx
+    //         .map(|idx| &source_file.as_str()[..idx])
+    //         .unwrap_or(source_file.as_str());
+    //     file.write_all(schema.as_bytes()).unwrap();
+    //
+    //     for line in errors.lines() {
+    //         writeln!(file, "// {line}").unwrap();
+    //     }
+    //     return;
+    // }
+
+    // panic_with_diff::panic_with_diff(&last_comment_contents, &errors)
```

Comment:
the commented out code should be removed here as well

Reply:
Right, it was a leftover